javers-core/src/main/java/org/javers/core/graph/ObjectNode.java

```diff
@@ -75,7 +78,11 @@ public GlobalId getReference(Property property){
 
     public Object getPropertyValue(Property property) {
         Validate.argumentIsNotNull(property);
-        return cdo.getPropertyValue(property);
+        try {
+            return cdo.getPropertyValue(property);
+        } catch (JaversGetterException e){
```

Comment:
średnio mi się podoba to połykanie wyjątku, jesteś pewny, że za IllegalArgumentException może się kryć tylko TYPE_MISSMATCH?